webapplication/aoi/models.py

```diff
@@ -69,6 +72,7 @@ class Component(models.Model):
     success = models.BooleanField(default=False, verbose_name='Validation succeeded')
     additional_parameter = models.CharField(max_length=50, null=True, blank=True, verbose_name='Additional parameter')
     run_on_gpu = models.BooleanField(default=True, verbose_name='Whether GPU is needed for a component to run')
+    sentinel_image_type = models.IntegerField(choices=settings.SENTINEL_IMAGE_CHOICES, default=settings.SENTINEL_IMAGE_NONE_TYPE)
```

Comment:
I believe we should rely on required creds. E.g., if sentinel1_aws_creds_required then Sentinel-1 is required. We have an issue that remote executor doesn't require credentials. I'd better set required creds for remote executor even if they are not set.